scripts/jenkins-yoctobuild-build.sh

```diff
@@ -522,6 +522,12 @@ then
     cd "$WORKSPACE"
     source oe-init-build-env build-rpi3
 
+    if [ "$META_MENDER_REV" == "pyro" ]; then
```

Comment:
This won't work. It may be a pull request. Use the same method as already exists for pyro in this file.

Reply:
thanks